src/Nethermind/Nethermind.Core/Caching/MemCountingCache.cs

```diff
@@ -45,7 +45,7 @@ public MemCountingCache(int maxCapacity, string name)
         }
 
         [MethodImpl(MethodImplOptions.Synchronized)]
-        public byte[]? Get(KeccakKey key)
+        public byte[]? Get(ValueKeccak key)
```

Comment:
`in` modifier to one less copy?

Reply:
Error when implementing the interface which doesn't use `in`